tests/posix/common/src/clock.c

```diff
@@ -46,34 +46,16 @@ void test_posix_clock(void)
 
 void test_posix_realtime(void)
 {
-	/* Make sure the realtime and monotonic clocks start out the
-	 * same.  This is not true on posix and where there is a
-	 * realtime clock, so don't keep this code.
-	 */
-
 	int ret;
 	struct timespec rts, mts;
 	struct timeval tv;
 
-	printk("POSIX clock set APIs\n");
```

Comment:
why are you removing this?

Reply:
It seems weird to have additional printk's here especially when it isn't simply testing clock_set but also clock_get and other calls.